examples/js/animation/MMDAnimationHelper.js

```diff
@@ -945,6 +945,8 @@ THREE.MMDAnimationHelper = ( function () {
 			}
 
 			if ( this.currentTime < this.delayTime ) return false;
+			
+			if ((this.currentTime - this.delayTime) > this.audioDuration) return false;
```

Comment:
Please respect the code style. The line should look like this:
```js
if ( ( this.currentTime - this.delayTime ) > this.audioDuration ) return false;
```

Reply:
I have adjusted the code style, please check, Thanks.